common/reviews/api/api-extractor-model.api.md

```diff
@@ -934,6 +954,12 @@ export interface IResolveDeclarationReferenceResult {
     resolvedApiItem: ApiItem | undefined;
 }
 
+// @public
+export interface IResolveInheritedMembersResult {
+    errorMessage?: string;
+    resolvedMembers: ApiItem[];
```

Comment:
It seems a bit limiting to only show one error message, with no way to provide additional such as the associated `ApiItem`.  A more flexible approach might be like: (?)

```ts
let incompleteResult: boolean = false;
const result: ApiItem[] = findInheritedMembers({
  messageCallback: (message) => {
    console.log(message.text);
    incompleteResult = true;
  }
});
```

Reply:
The messages will have IDs like `message.id === ExtendsFromTypeAlias` or `message.id === MissingApiJson`.

Maybe we should also have `message.incompleteMembers`